xml/Sector.xml

```diff
@@ -872,7 +872,17 @@
 				<narrative>Women’s equality organisations and institutions</narrative>
 			</name>
 			<description>
-				<narrative>Support for institutions and organisations (governmental and non-governmental) working for gender equality and women's empowerment.</narrative>
+				<narrative>Support for institutions and organisations (governmental and non-governmental) working for gender equality and women’s empowerment.</narrative>
+			</description>
+			<category>151</category>
+		</codelist-item>
+		<codelist-item>
+			<code>15180</code>
+			<name>
+				<narrative>Ending violence against women and girls</narrative>
+			</name>
+			<description>
+				<narrative>Support to programmes designed to prevent and eliminate all forms of violence against women and girls/gender-based violence. This encompasses a broad range of forms of physical, sexual and psychological violence including but not limited to: intimate partner violence (domestic violence); sexual violence; female genital mutilation/cutting (FGM/C); child, early and forced marriage; acid throwing; honour killings; and trafficking of women and girls. Prevention activities may include efforts to empower women and girls; change attitudes, norms and behaviour; adopt and enact legal reforms; and strengthen implementation of laws and policies on ending violence against women and girls, including through strengthening institutional capacity. Interventions to respond to violence against women and girls/gender-based violence may include expanding access to services including legal assistance, psychosocial counselling and health care; training personnel to respond more effectively to the needs of survivors; and ensuring investigation, prosecution and punishment of perpetrators of violence.</narrative>
```

Comment:
This is actually a new codelist item (15180) – it’s just presented in a confusing way here.